keps/sig-windows/20190103-windows-node-support.md

```diff
@@ -111,20 +123,29 @@ As of 29-11-2018 much of the work for enabling Windows nodes has already been co
 
 ### What will never work (Note that some features are plain unsupported while some will not work without underlying OS changes)
 - Certain Pod functionality
-    - Privileged containers and other Pod security context privilege and access control settings
+    - Privileged containers
+    - Pod security context privilege and access control settings. Any Linux capabilities or any POSIX capabilities are not supported (this includes SELinux, AppArmor, Seccomp, etc)
     - Reservations are not enforced by the OS, but overprovisioning could be blocked with `--enforce-node-allocatable=pods` (pending: tests needed)
     - Certain volume mappings
       - Single file & subpath volume mounting
```

Comment:
Will single-file mapping become possible? This comment makes it sound like it will:
https://github.com/kubernetes/kubernetes/issues/70189#issuecomment-432837906

Reply:
My understanding is that this will be supported with containerd (WIP), but not possible right now with the docker integration (which is used in GA).

Reply:
you are correct @yujuhong